src/sql/parts/query/editor/messagePanel.ts

```diff
@@ -145,27 +147,37 @@ export class MessagePanel extends ViewletPanel {
 		let hasError = false;
 		if (isArray(message)) {
 			hasError = message.find(e => e.isError) ? true : false;
+			message.forEach(resultMessage => this.countMessageLines(resultMessage));
```

Comment:
small thing, this could be changed to a reduce method and remove the property from the object